front/src/components/Navigation/Toolbar/Toolbar.css

```diff
@@ -0,0 +1,58 @@
+.Toolbar {
+    width: 100%;
+    position: fixed;
+    /* background: rgb(205, 143, 86); */
+    background: linear-gradient(
+        120deg,
+        rgba(205, 143, 86, 1) 20%,
+        rgba(217, 235, 52, 1) 98%
+    );
+    background-size: cover;
+    top: 0;
+    left: 0;
+    align-items: center;
+    box-sizing: border-box;
+    z-index: 90;
+    text-align: center;
+}
+
+.Toolbar nav {
+    height: 100%;
+    width: 100%;
+    /* background-color: #6b9ff2; */
+    background-color: rgba(214, 193, 137, 0.96);
+    align-items: center;
+}
+
+.Logofs {
+    margin-left: 20px;
+}
+
+@media (min-width: 600px) {
+    .Logofs {
+        height: 140px;
+        padding-top: 15px;
+        padding-bottom: 15px;
+    }
+    .Toolbar {
+        justify-content: space-evenly;
+    }
+}
+@media (max-width: 599px) {
+    .DesktopOnly {
+        display: none;
+    }
+    .Logofs {
+        height: 50px;
```

Comment:
Remove it - changing element's height while it's not being displayed is redundant